test/http_generic/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3.0'
+
+services:
+  nginx:
+    image: nginx:stable-alpine
+    ports:
+      - 8080
+    healthcheck:
+      test: ["CMD-SHELL", "curl localhost:8080/test"]
+      interval: 1s
+      timeout: 30s
+    volumes:
+      - ./http-basic-auth.conf:/etc/nginx/conf.d/default.conf:ro
+      - ./basic_auth.htpasswd:/etc/nginx/.htpasswd:ro
+
+  secretless:
+    build:
+      context: ../..
+    environment:
```

Comment:
These envvars aren't use anywhere as far as I can tell. If not let's get rid of them

Reply:
I'll try it and see we'll see if anything breaks :)